spec/factories.rb

```diff
@@ -1,9 +1,23 @@
 FactoryGirl.define do
+
+  sequence(:email) { |n| "#{n}@test.com" }
+  sequence(:phone_number) { |n| "#{n}123456789"}
```

Comment:
@riverswb Danger, Will Robinson!  How long can a phone number be?  10 digits?  11?  This factory is going to make some long ones if we're keeping them unique and keep upping the sequence. What about using the sequence in the last 4 digits or something?  How about this...
`sequence(:last_four) { |n| n.to_s.rjust(4, "0") }
`phone_number { "123456" + last_four }`

I'm not 100% sure that will work as is, but the thought is that it would produce 1234560000, 1234560001... etc...